test/v2/models/request/createAmendHistoricNonFhlUkPropertyAnnualSubmission/CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionRequestBodySpec.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.request.createAmendHistoricNonFhlUkPropertyAnnualSubmission
+
+import play.api.libs.json.{ JsValue, Json }
+import support.UnitSpec
+import v2.models.request.common.ukPropertyRentARoom.UkPropertyAdjustmentsRentARoom
+
+class CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionRequestBodySpec extends UnitSpec {
+
+  val requestBody: CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionRequestBody =
+    CreateAmendHistoricNonFhlUkPropertyAnnualSubmissionRequestBody(
+      annualAdjustments = Some(
+        HistoricNonFhlAnnualAdjustments(
```

Comment:
Can you refactor to separate out `HistoricNonFhlAnnualAdjustments` to be it's own value and pass it in. Makes the code much easier to read. Same for annual allowances